metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -1165,6 +1165,9 @@ def __init__(self, looker_api: LookerAPI):
         self.looker_api_wrapper = looker_api
 
     def get_by_id(self, id_: str) -> Optional[LookerUser]:
+        if not id_:
+            return None
+
```

Comment:
Is this intentional? Seems like a good change though

Reply:
nothing added to commit but untracked files present (use "git add" to track)
 datahub/datahub-integrations-service on  no-slacking-hack [$?] is 📦 v0.1.0 via 🐍 v3.10.8 (acryl-datahub-integrations-py3.10)  ❯
gl
remote: Enumeratin

Reply:
yup meant to put in a separate PR but it's fine here too